kod/object/item/passitem/spcgift2.kod

```diff
@@ -40,6 +40,8 @@ classvars:
 
    viValue_average = 30
 
+   vrGift_open_wav = gift_open_wav
```

Comment:
I followed the example set forth in a few other classes, including `food.kod` and it's child classes.

However I now realize this is pointless.  It works directly when called. I don't think we need the class variable at all, as you mention.  I changed it to call the only once initialized resource `gift_open_wav` since the unwrap sound will be the same across all 3 gift classes.